index.html

```diff
@@ -0,0 +1 @@
+<dev>Hello World <dev>
```

Comment:
Allow this indext file as sample test file of what the dev team is going to be doing.

Reply:
Thank you